aprs_service_registry/health_checker.py

```
@@ -0,0 +1,453 @@
+        )
+        tx.send(packet, direct=True)
+
+        # Start consumer in a thread to receive response
+        client = APRSDClient()
+
+        def consume():
+            try:
+                client.consumer(rx_callback, raw=False, blocking=True)
+            except StopIteration:
+                pass
+            except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)